koa-crud/src/use-cases/promo-enrollments/enroll-to-promo.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  UseCase,
+  MembersStore,
+  PromosStore,
+  PromoTemplate,
+  PromoStatus,
+  PromoEnrollmentsStore,
+} from '../../types';
+
+const enrollToPromo = ({
+  membersStore,
+  promosStore,
+  R,
+  promoEnrollmentsStore,
+}: {
+  membersStore: MembersStore;
+  promosStore: PromosStore;
+  R;
+  promoEnrollmentsStore: PromoEnrollmentsStore;
+}): UseCase<boolean> => {
+  const underscoreToCamelCase = (str) => {
+    return str.toLowerCase().replace(/_+(\w|$)/g, ($$, $1) => {
+      return $1.toUpperCase();
+    });
+  };
```

Comment:
You can also use this, https://github.com/sindresorhus/camelcase?

Reply:
noted sir